spec/lib/example_campaign_generator_spec.rb

```diff
@@ -75,6 +81,34 @@
     expect(json).to eq(expected_flu_json)
   end
 
+  it "generates the expected empty pilot campaign json" do
+    json = nil
+    # Patient's dob is generated as an age calculated from when the fixture json
+    # was generated, so we freeze time to the same day it was generated to get
+    # the same random ages.
+    #
+    # When the fixture json is regenerated, e.g. when the generater is updated,
+    # the time here will need to be changed to match the day when it was
+    # generated.
+    #
+    # To regenerate:
+    #     rails generate_example_campaign \
+    #       seed=44 \
+    #       presets=empty_pilot \
+    #       username="Nurse Flo" > db/sample_data/example-pilot-campaign.json
+    Timecop.freeze(2024, 2, 8, 12, 0, 0) do
```

Comment:
We should probably combine this + above ones in a script so it's 1 manual step not 3...

Reply:
Added commit